packages/actor-http-native/lib/Requester.ts

```diff
@@ -13,7 +13,7 @@ const { https } = require('follow-redirects');
 // Decode encoded streams with these decoders
 const DECODERS = { gzip: zlib.createGunzip, deflate: zlib.createInflate };
 
-export default class Requester {
+export class Requester {
```

Comment:
This change will also have to be applied to the browser version: https://github.com/comunica/comunica/blob/master/packages/actor-http-native/lib/Requester-browser.ts#L14

These two always need to stay in sync.

Reply:
Actually, instead of modifying this, why not just make `convertFetchHeadersToHash` a member method to this class?